.github/workflows/bench.yml

```diff
@@ -49,17 +53,48 @@ jobs:
         cargo -V || echo "no cargo"
 
     - uses: Swatinem/rust-cache@v2
-    
+
+    # Compute cache key and save to a temporary file.
+    #
+    # We compute the cache key based on the solved install plan instead of just
+    # hashing the `.cabal` file, since there are many kinds of changes that will
+    # cause `.cabal` to change (e.g. adding new source modules).
```

Comment:
Do such changes not always result in changes to the install plan? I'd think they would.

Reply:
The install plan only specifies the dependent packages that need to be there for the build. For spectrometer, it just lists the various spectrometer targets like fossa, pathfinder, the library, etc. Adding a new source module won't change the build plan. Adding a new 3rd party dep, for instance, would.

Reply:
Ah, gotcha 👍🏻